pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -207,6 +210,36 @@ public Set<NamespaceBundle> getOwnedServiceUnits() {
                     var bundle = entry.getKey();
                     return getNamespaceBundle(pulsar, bundle);
                 }).collect(Collectors.toSet());
+        // Add heartbeat and SLA monitor namespace bundle.
+        NamespaceName heartbeatNamespace = NamespaceService.getHeartbeatNamespace(brokerId, pulsar.getConfiguration());
```

Comment:
Why do we need to add these static bundles(bundles that are not stored in BSC) for `getOwnedServiceUnits`?

Reply:
Yes, for `getOwnedServiceUnits` it needs to know the heartbeat and SLA monitor the ownership. it just keeps the behavior the same as previously.